digits/templates/layout.html

```diff
@@ -77,15 +77,8 @@
                return base;
             }
             return baseName(suggestion.value);
-        }
-    });
-    $(".autocomplete_path").removeAttr("autocomplete");
-    $(".autocomplete_path").on('keyup',function(e){
-      // If path is relative, enforce absolute
-      var isRelative = e.target.value[0] != '/' && e.target.value[0] != '\\';
-      if (e.target.value.length > 0 && isRelative) {
-        e.target.value = "/" + e.target.value;
-      }
+        },
+        minChars: 0
```

Comment:
The fix seems not working well.  If I type /working/, I could see a list of subfolders and files under c:/working/  (my digits is installed on drive c:).
However, if I type c:/working/, there is no list shown.  At the same time, browser showed two requests, path?query=c and path?query=c%3A.  Nothing else.